<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportsConfig</class>
 <widget class="QWidget" name="ReportsConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>886</width>
    <height>610</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Report Contents</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCheckBox" name="TitlePageCheckbox">
            <property name="text">
             <string>Title Page</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="TableOfContentsCheckbox">
            <property name="text">
             <string>Table Of Contents</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="ExecutiveSummaryCheckbox">
            <property name="text">
             <string>Executive Summary</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="DetailedReportCheckbox">
            <property name="text">
             <string>Detailed Report</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="RemediationReportCheckbox">
            <property name="text">
             <string>Remediation Report</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="HistoricalDataReportCheckbox">
            <property name="text">
             <string>Historical Data Report</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Audit History To Include:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QCheckBox" name="IncludeOnlyMatchingCheckbox">
              <property name="text">
               <string>Include only audits matching same import machine list</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="IncludeOnlyCompletedCheckbox">
              <property name="text">
               <string>Include only completed audits</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QRadioButton" name="LastAuditRadio">
            <property name="text">
             <string>Comparison of most recent audit and previous audit</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="PreviousAuditsRadio">
            <property name="text">
             <string>Comparison of most recent audit and multiple previous audits</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="leftMargin">
             <number>16</number>
            </property>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Number of previous audits to include:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="NumberOfPreviousEdit">
              <property name="maximumSize">
               <size>
                <width>30</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="inputMask">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QRadioButton" name="ReportsInDateRangeRadio">
            <property name="text">
             <string>Comparison of audits in date range</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <property name="leftMargin">
             <number>16</number>
            </property>
            <item>
             <widget class="QCheckBox" name="FromCheckbox">
              <property name="text">
               <string>From:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateEdit" name="FromDate">
              <property name="calendarPopup">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="ToCheckbox">
              <property name="text">
               <string>To:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateEdit" name="ToDate">
              <property name="calendarPopup">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QRadioButton" name="SpecificReportsRadio">
            <property name="text">
             <string>Comparison of specific audits</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <property name="leftMargin">
             <number>16</number>
            </property>
            <item>
             <widget class="QPushButton" name="SelectHistoricalDataButton">
              <property name="text">
               <string>Select Historical Data...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTreeWidget" name="HistoricalDataTree">
              <property name="rootIsDecorated">
               <bool>false</bool>
              </property>
              <property name="uniformRowHeights">
               <bool>true</bool>
              </property>
              <property name="columnCount">
               <number>2</number>
              </property>
              <attribute name="headerStretchLastSection">
               <bool>true</bool>
              </attribute>
              <column>
               <property name="text">
                <string notr="true">1</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string notr="true">2</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Report Customizations</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="2" column="0">
           <widget class="QCheckBox" name="ReportNameCheckbox">
            <property name="text">
             <string>Report Name</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="ReportNameEdit">
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string/>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="ReportDateCheckbox">
            <property name="text">
             <string>Report Date</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QDateTimeEdit" name="ReportDateEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QCheckBox" name="L0phtCrackLogoCheckbox">
            <property name="text">
             <string>L0phtCrack Logo</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QCheckBox" name="CompanyNameCheckbox">
            <property name="text">
             <string>Company Name</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="CompanyNameEdit"/>
          </item>
          <item row="6" column="0">
           <widget class="QCheckBox" name="CompanyLogoCheckbox">
            <property name="text">
             <string>Company Logo</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QLineEdit" name="CompanyLogoPath"/>
                </item>
                <item>
                 <widget class="QPushButton" name="BrowseCompanyLogoButton">
                  <property name="text">
                   <string>Browse...</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="lc7reports.qrc">:/lc7/unspecified.png</pixmap>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="12" column="0">
           <widget class="QCheckBox" name="PreparerNameCheckbox">
            <property name="text">
             <string>Preparer Name</string>
            </property>
           </widget>
          </item>
          <item row="12" column="1">
           <widget class="QLineEdit" name="PreparerNameEdit"/>
          </item>
          <item row="13" column="0">
           <widget class="QCheckBox" name="PreparerLogoCheckbox">
            <property name="text">
             <string>Preparer Logo</string>
            </property>
           </widget>
          </item>
          <item row="13" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QLineEdit" name="PreparerLogoPath"/>
                </item>
                <item>
                 <widget class="QPushButton" name="BrowsePreparerLogoButton">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>Browse...</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <spacer name="verticalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="lc7reports.qrc">:/lc7/unspecified.png</pixmap>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="14" column="0">
           <widget class="QCheckBox" name="ConfidentialityStatementCheckbox">
            <property name="text">
             <string>Confidentiality Statement</string>
            </property>
           </widget>
          </item>
          <item row="14" column="1">
           <widget class="QTextEdit" name="ConfidentialityStatementEdit">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>80</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
         <zorder>ConfidentialityStatementCheckbox</zorder>
         <zorder>ConfidentialityStatementEdit</zorder>
         <zorder>CompanyNameCheckbox</zorder>
         <zorder>CompanyNameEdit</zorder>
         <zorder>CompanyLogoCheckbox</zorder>
         <zorder>ReportNameCheckbox</zorder>
         <zorder>ReportDateCheckbox</zorder>
         <zorder>ReportNameEdit</zorder>
         <zorder>ReportDateEdit</zorder>
         <zorder>PreparerNameCheckbox</zorder>
         <zorder>L0phtCrackLogoCheckbox</zorder>
         <zorder>PreparerLogoCheckbox</zorder>
         <zorder>PreparerNameEdit</zorder>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="NoDataLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>Before you can generate a report, you must first have some audit data. Perform an audit or multiple audits to generate the report data.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="lc7reports.qrc"/>
 </resources>
 <connections/>
</ui>
